core/types/bid.go

```diff
@@ -70,12 +79,13 @@ func (b *BidArgs) ToBid(builder common.Address, signer Signer) (*Bid, error) {
 
 // RawBid represents a raw bid from builder directly.
 type RawBid struct {
-	BlockNumber uint64          `json:"blockNumber"`
-	ParentHash  common.Hash     `json:"parentHash"`
-	Txs         []hexutil.Bytes `json:"txs"`
-	GasUsed     uint64          `json:"gasUsed"`
-	GasFee      *big.Int        `json:"gasFee"`
-	BuilderFee  *big.Int        `json:"builderFee"`
+	BlockNumber   uint64          `json:"blockNumber"`
+	ParentHash    common.Hash     `json:"parentHash"`
+	Txs           []hexutil.Bytes `json:"txs"`
+	NonRevertible []common.Hash   `json:"nonRevertible"`
```

Comment:
prefer Revertible, for less request size, also consistent with sendBundle

Reply:
Actually,  NonRevertible is better, because tx from searcher is less I believe, if you check the MEV explorer.

Reply:
ok, but NonRevertible=>UnRevertible